CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-07-02
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* Windows 2019 will be retired in Kubernetes v1.33 and above (ETA March 2026). Customers should [upgrade to Windows 2022](https://learn.microsoft.com/azure/aks/upgrade-windows-2019-2022).
+* Kubernetes 1.24 is being deprecated end of July 2023. From Kubernetes 1.25 the default cgroups implementation on AKS nodes will be cgroupsv2. Older versions of Java, .NET and NodeJS do not support memory querying v2 memory constraints and this will lead to out of memory (OOM) issues for workloads. Please test your applications for cgroupsv2 compliance.
+* A [known issue](https://github.com/Azure/AKS/issues/3718) in Kubernetes version 1.24 is causing name resolution failures in Windows pods. Customers experiencing this issue should upgrade their cluster to Kubernetes version 1.25.
+
+### Release notes
+
+* Preview Features
+  * Added Prometheus scraping annotations to App Routing add-on.
+  * Support for Azure DNS private zones in the App Routing add-on.
+  * Support for changing pod CIDR for bring your own CNI plugin.
+
+* Behavior Changes
+  * The default OS disk type for non-ephemeral OS disks is now Standard SSD.
+  * Enable on-call engineers to access Open Service Mesh (OSM) CRDs for debugging.
+
+* Bug Fixes
```

Comment:
Anything customers need to do to get these fixes?

Reply:
Not for these 2 fixes.